confluent_server/bin/confluent_selfcheck

```diff
@@ -259,6 +260,15 @@ if __name__ == '__main__':
                     mac = rsp[key].get('value', None)
                     if mac:
                         macok = True
+            #adding new code to check if the response is something like net.<something>switch
+            for key in rsp:
+                if key.startswith('new.')and key.endswith('switch'):
+                    switch_value = rsp[key].get('value',None)
+                    if switch_value == 'true':
+                        if args.node in valid_nodes:
```

Comment:
The 'switch_value' would be a switch name, not the word 'true', we are checking for that to be a valid node, not that it is 'true'